### PR TITLE
Beta builds

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a879f104fa1f21b1102f5d7aa1253a5820cf5d59-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a879f104fa1f21b1102f5d7aa1253a5820cf5d59-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a879f104fa1f21b1102f5d7aa1253a5820cf5d59-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a879f104fa1f21b1102f5d7aa1253a5820cf5d59-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a43120d7c999f574dc1f8da077931fe114c866ee-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a43120d7c999f574dc1f8da077931fe114c866ee-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a43120d7c999f574dc1f8da077931fe114c866ee-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-a43120d7c999f574dc1f8da077931fe114c866ee-x86-5.apk))

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +26,9 @@ script:
   - echo "travis_fold:start:tx-pull"
   - ./scripts/ci/pull_translations.sh
   - echo "travis_fold:end:tx-pull"
-  - docker run -v $(pwd):/usr/src/qfield -e "BUILD_FOLDER=build-${ARCH}" -e "ARCH=${ARCH}" -e "STOREPASS=${STOREPASS}" -e "KEYNAME=${KEYNAME}" -e "KEYPASS=${KEYPASS}" -e "VERSION=${TRAVIS_TAG}" opengisch/qfield-sdk:${QFIELD_SDK_VERSION} /usr/src/qfield/scripts/docker-build.sh
+  - APP_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "QField"; else echo "QField Beta ${TRAVIS_PULL_REQUEST}"; fi)
+  - PKG_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "qfield"; else echo "qfield_beta"; fi)
```
Comment (3nids) [inline .travis.yml]:
```suggestion
  - PKG_NAME=$(if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then echo "qfield"; else echo "qfield_beta_${TRAVIS_PULL_REQUEST}"; fi)
```

allow to have several apps?

Comment (m-kuhn) [reply .travis.yml]: I thought about that too.
I think it's usually enough to test one PR at a time and as a bonus you don't end up with 20 packages installed in parallel (and having to uninstall them all manually).

Comment (3nids) [reply .travis.yml]: Fine with me, we can still fix it up later.

**Comment by 3nids** [comment]
Also, maybe in icon app with beta written would be cool. And for later, a banner in the GUI.

**Comment by signedav** [comment]
If it's easy to do it would be nice to have another logo on the tablet menu: https://github.com/opengisch/QField/blob/1dc593c2cd6bb182f30767a95bf207cbb5b31108/images/icons/qfield-testlogo.svg 

Could be something else of course as well.

**Comment by signedav** [comment]
Oh haven't seen your comment @3nids before...

**Comment by m-kuhn** [comment]
Let's see if that works

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-4ee8b8002725a47d970d05f9ac4ba7aa51eb89ab-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-4ee8b8002725a47d970d05f9ac4ba7aa51eb89ab-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-4ee8b8002725a47d970d05f9ac4ba7aa51eb89ab-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-4ee8b8002725a47d970d05f9ac4ba7aa51eb89ab-x86-5.apk))

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-710417ac8506e5874f34bcb26265625ab4cf7f4a-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-710417ac8506e5874f34bcb26265625ab4cf7f4a-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-710417ac8506e5874f34bcb26265625ab4cf7f4a-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-610-710417ac8506e5874f34bcb26265625ab4cf7f4a-x86-5.apk))

**Pull request merged**